docs/HowTo/Get-Started/Build-From-Source.md

```diff
@@ -9,7 +9,7 @@ description: Building EthSigner from source code
 * [Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
 
 !!!important
-    EthSigner requires Java 11+; earlier versions are not supported.
+    EthSigner requires Java 11 are later releases.
```

Comment:
This edit doesn't make sense - I think it's meant to be the same as the one below.

Reply:
It was a typo. Fixed now.